steambox_nvidia/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM steambox
+MAINTAINER Jereme Corrado "jereme@zoion.net"
+
+USER root
+
+RUN apt-get install -y debian-archive-keyring
+RUN echo "deb http://ftp.debian.org/debian/ jessie-backports main contrib non-free" > /etc/apt/sources.list.d/jessie-backports.list
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
+    apt-get install -y -t jessie-backports nvidia-driver
```

Comment:
@jcorrado I still have the concern that this is installing a Debian-specific driver.  It may work ok for people running vanilla Debian host machines, but I'm not as sure about people running Ubuntu, Fedora, Mint, etc.

I'm going to accept this as-is though because having  Debian+Nvidia support is better than having no Nvidia support at all. :)

It can always be addressed in a follow-on changeset if anyone has an issue.